### DISCUSSION TITLE
navigating/scrolling frames

### DISCUSSION THREAD

**Discussion by grault** [qutebrowser/qutebrowser#6621] [Q&A] [answered]
I'm working with pages that have multiple independently scrollable frames. Eg Slack, Jira..

There's this config.content.frame_flattening setting, which I'm currently looking at as workaround.

Question is: how am I focusing frames? So that pgdown and pgup interacts with that frame.

**Comment by toofar** [comment, marked as answer]
Using the mouse!
There is a PR to add better support for frames https://github.com/qutebrowser/qutebrowser/pull/3691
So, it's not supported currently. Go and :+1: that PR and the linked issue I suppose.
